.jenkins/projectBuilder.Jenkinsfile

```diff
@@ -54,6 +58,10 @@ projectBuilderV5 (
                 GOOGLE_CLIENT_ID: '@vault(secret/configs/upgrade/${environment}/GOOGLE_CLIENT_ID)',
                 GOOGLE_SERVICE_ACCOUNT_ID: '@vault(secret/configs/upgrade/${environment}/GOOGLE_SERVICE_ACCOUNT_ID)',
             ],
+            automatedBranchBuilds: [
+                "main",
```

Comment:
what does main refer to, the actual `main` branch?

Reply:
should be dev